src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -201,7 +201,14 @@ trait DeploymentFormats {
         JsArray(xs.to[Seq].map(b => JsNumber(b.toInt)))
       }
     )
-  implicit lazy val GroupFormat: Format[Group] = Json.format[Group]
+  implicit lazy val GroupFormat: Format[Group] = (
+    (__ \ "id").format[PathId] ~
+    (__ \ "apps").formatNullable[Set[AppDefinition]].withDefault(Set.empty) ~
```

Comment:
Could we define the defaults instead in the `Group` companion object, to mirror how it is done for `AppDefinition`?